SEP-0009.md

```diff
@@ -0,0 +1,106 @@
+correctness.
+
+This SEP therefore tries to focus on when the SunPy core package will be
+released rather than placing any strict limitations on changes and API
+stability.
+
+# Detailed Description
+
+## Release Schedule
+
+This SEP proposes that the SunPy core package will be released at least twice a
+year, initially a release in the months of May and November have been chosen to
+place the SunPy release shortly after that of the Astropy core package. The
+decision to make a release and the specific timing of releases are at the
+discretion of the lead developer, and changes to the months of the releases can
+be made without modifying this SEP to follow any pertinent release cycles of
+upstream packages or any other reason.
+
+Future changes to the LTS release cycle should be informed by upstream package
+development. It is important that upstream packages versions are supported for
+the lifetime of a SunPy LTS.
+
+## Supporting Releases with bug fixes
+
+This SEP proposes that there be two categories of SunPy release, a LTS (Long
+Term Support) release and a normal or non-LTS release. The LTS release will have
+an extended length of bug fixes provided by the SunPy developers. We propose
+that the first release in a year be a non-LTS release, which will be supported
+for 6 months (until the next release), the second release in a year will be an
+LTS release and will be supported for 12 months (until the next LTS release).
+
+The objective of this support cycle is to balance the desire to provide a
+release cycle for people who wish to experience minimal changes to SunPy with
+minimizing the maintenance burden on the SunPy developers. It is feasible that
+we could extend the support cycle of the LTS release to longer than 12 months,
+to give users an overlap period where they can upgrade. However, at this time,
+supporting more than two releases simultaneously is beyond the
+resources of the project.
+
+## Release Numbering
+
+In the numbering of SunPy releases we wish to covey if the release is an LTS
+release or not. We therefore propose the following versioning system.
+
+X.Y.z
+
+Where the three components have the following meanings:
+
+* X. is the LTS version number, this will be incremented with every LTS.
+* Y. is the release counter, this will be 0 for LTS releases and increment for each intermediate release.
+* z. is the bug fix number, and is to be incremented for any bug fix releases.
+
+A typical release sequence, following this scheme, would look like this:
+
+* 1.0.0 is an LTS release, released November 2019.
+* 1.0.1 is a bugfix to the LTS release, released December 2019.
+* 1.1.0 is a short support release, released May 2020.
+* 1.0.2 and 1.1.1 are bugfix releases, released June 2020, to fix a current bug and backport the fix to the LTS version.
+* 1.2.0 is an unplanned short support release, released August 2020
+* 2.0.0 is an LTS release, released November 2020.
+* And so on...
+
+## Deprecations and Documentation
```

Comment:
what is the relationship between version numbers and deprecation warnings?

Reply:
Only that we should deprecate functionality for *at least* one release before we remove it normally.